apps/gallery/stories/components/wui-icon-box.stories.ts

```diff
@@ -0,0 +1,57 @@
+import type { Meta } from '@storybook/web-components'
```

Comment:
This entire file (wui-icon-box.stories.ts) file needs to move to `stories/composites`